exporter/templates/cat1/r3/_code.xml

```diff
@@ -1,8 +1,10 @@
 <{{- valueOrDefault .TagName "code" -}}
-  {{- if hasPreferredCode .PreferredCode}} code='{{- .PreferredCode.Code}}' codeSystem='{{- oidForCodeSystem .PreferredCode.CodeSystem}}' {{.RenderExtraContent}}
+  {{- if hasPreferredCode .PreferredCode}}
+    {{- if .MapDataCriteria.DataCriteria.Negation}} nullFlavor="NA" {{.RenderExtraContent}}{{- else}} code="{{- .PreferredCode.Code}}" codeSystem="{{- oidForCodeSystem .PreferredCode.CodeSystem}}" {{.RenderExtraContent}}
```

Comment:
Why isn't `{{.RenderExtraContent}}` placed at the end of the if-else? It seems to be used regardless of which flow is reached.